### PR TITLE
zdbsp: init at 1.19

### PR DESCRIPTION
**Pull request by esoeylemez**

### CODE REVIEW

File: pkgs/games/zdoom/zdbsp.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  installPhase = ''
+    install -Dm755 zdbsp $out/bin/zdbsp
+  '';
+
+  meta = with stdenv.lib; {
+    description = "ZDoom's internal node builder for DOOM maps";
+    homepage = "https://zdoom.org/wiki/ZDBSP";
+    license = licenses.gpl2Plus;
+    maintainers = with maintainers; [ertes];
+    platforms = platforms.linux;
```
Comment (rasendubi) [inline pkgs/games/zdoom/zdbsp.nix]: Any reason this package is linux-only?

**Review by rasendubi — APPROVED** [review]
Looks good for me. Builds fine on NixOS with sandboxing.

**Comment by Mic92** [comment]
@esoeylemez Could please rename the commit message to `zdbsp: init at 1.19`. This how we usually word commits for new packages.

**Comment by esoeylemez** [comment]
Renamed and rebased.

File: pkgs/games/zdoom/zdbsp.nix (hunk 1 above)
Comment (esoeylemez) [inline pkgs/games/zdoom/zdbsp.nix]: Only that I haven't tested it on other platforms (and because zdoom is marked as Linux-only, too).

**Pull request merged**